app/services/trending_service.rb

```diff
@@ -75,6 +77,24 @@ def trending_key(uid = nil)
     key
   end
 
+  def trending_category_key(category_id)
+    ns = namespace
+    ns = ns.table_name if enrichable?
+    key = "trending:#{ns}"
+    key += ":category:#{category_id}"
+    key
+  end
+
+  def handle_trending_categories(id, weight = 1.0)
+    return unless %w[Anime Manga Drama].include?(namespace.name)
+    categories = namespace.includes(:categories).find_by(id: id).categories
+    categories.each do |category|
+      key = trending_category_key(category.id)
+      update_score(key, id, weight)
+      trim(key, limit: NETWORK_LIMIT) if rand < TRIM_PROBABILITY
```

Comment:
Make sense to me, although i'm sure what would be appropriate to set it to. Should we keep it at 20?